scss/modules/_feedback.scss

```diff
@@ -1,14 +1,17 @@
 // Feedback module
 
 .feedback {
-  @include transition(bottom .5s);
   @include u-font-color($cream);
   @include u-form-color($navy, $primary-contrast, $ivory, 'primary', $primary-contrast);
   background-color: $navy;
-  bottom: 0;
-  padding: 1rem;
+  display: block;
```

Comment:
Properties should be ordered background-color, bottom, display, left, overflow-y, padding, position, right, top, z-index